pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -374,7 +374,7 @@ def run_evaluation(self, test_mode: bool = False):
 
         # cap max batches to 1 when using fast_dev_run
         if self.fast_dev_run:
-            max_batches = [1]
+            max_batches = 1
```

Comment:
```suggestion
            max_batches = [1] * len(dataloaders)
```
Let's make it explicit? (i know an int works too)

Reply:
Ok, done.